pocs/utils/rs232.py

```diff
@@ -12,19 +12,31 @@
 
 
 class SerialData(PanBase):
-
     """
     Main serial class
     """
 
-    def __init__(self, port=None, baudrate=115200, threaded=True, name="serial_data"):
+    def __init__(self,
+                 port=None,
+                 baudrate=115200,
+                 threaded=None,
```

Comment:
I'm temped to say we should just add a `*args, **kwargs` and remove the `threaded=None`.

Reply:
I'd prefer to rip out the param as soon as the PACE code is merged in here, as that way we don't accidentally accept bogus params.